src/Elastic.CommonSchema.Serilog/ECSJsonFormatter.cs

```diff
@@ -0,0 +1,22 @@
+using System.IO;
+using System.Text;
+using Serilog.Events;
+using Serilog.Formatting;
+using Utf8Json.Resolvers;
+
+namespace Elastic.CommonSchema.Serilog
+{
+    /// <summary>
+    /// Writes the log event using Elasticsearch Common Schema JSON format
+    /// </summary>
+    public class ECSJsonFormatter : ITextFormatter
+    {
+        public void Format(LogEvent logEvent, TextWriter output)
+        {
+            var ecsEvent = LogEventConverter.ConvertToEcs(logEvent);
+            var bytes = Utf8Json.JsonSerializer.Serialize(ecsEvent, StandardResolver.ExcludeNull);
```

Comment:
Is there a way we can serialize directly to the writer?

Reply:
We could, although my preference here would be to explore some kind of code-gen from the spec, as its basically handle-turning the object instance through the `.Write()` functions on the TextWriter. I will look into this. I do wonder how much faster this would be compared to UTF8Json given the way it optimises the serialisation...

Reply:
Given the next release is likely to be a beta, how do we feel about leaving in UTF8Json for now and implementing the writer later?

Reply:
Thats fine but in the long run part of the value of the logger is to make sure certain keys are in order.

https://github.com/elastic/ecs-logging-java#why-ecs-logging
> Decently human-readable JSON structure 
The first three fields are always @timestamp, log.level and message. It's also possible to 
 format stack traces so that each element is rendered in a new line.